acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
@er0ck Is the discrepancy that @rlinehan brought up with `#{testdir}/environments` vs. `master.puppet('master')['codedir']` in the `environment_scenario-non_existent.rb` test also a problem though?  I wouldn't be able to explain why the `environment_scenario-non_existent.rb` test would pass for you without fixing up this discrepancy if that were the case, but do you disagree that this is a problem in the test code that should be addressed?  If you agree that it is a problem that should be fixed, do you think it would make sense to do that as part of this PR?